03-task-manager/starter/quizAnswers.txt

```diff
@@ -0,0 +1,12 @@
+1.Quickly explain how middleware is used in an Express application.
+    Middleware in Express is a function that handles the request-response cycle and performs tasks such as validation, authentication, data parsing, etc.
+    It can be added to an application using app.use() or app.route() and executed in a sequence.
+
+2.What does CRUD stand for?
+    CRUD stands for "Create, Read, Update, Delete."
```

Comment:
Yes that's right. To elaborate a bit more:

> Create involves creating a new record or data entry in the system.  
> Read  involves retrieving or reading the data from the system.  
> Update  involves modifying or updating the existing data in the system.  
> Delete involves deleting or removing the data from the system.  
>  
> 
> These are important because they are the most common operations that a backend API with a data store will need to do. They usually correspond with the HTTP methods POST, GET, PUT or PATCH, and DELETE.